Makefile

```diff
@@ -39,11 +39,11 @@ tools.pre-commit.install.Darwin:
 	@brew install pre-commit
 	@echo "Pre-commit installation complete"
 
-tools.pre-commit.setup: tools.airbyte-ci.install tools.pre-commit.install.$(OS) tools.git-hooks.clean ## Setup pre-commit hooks
+tools.git-hooks.install: tools.airbyte-ci.install tools.pre-commit.install.$(OS) tools.git-hooks.clean ## Setup pre-commit hooks
```

Comment:
This name is paired with the related `tools.git-hooks.clean` above in this file.

Reply:
Is this called implicitly as part of other tasks?

Reply:
ah I see